src/platform/site-wide/sass/merger.scss

```diff
@@ -614,20 +608,74 @@ img[data-srcset] {
 
 .usa-alert-full-width {
   background-color: $color-gray-lightest;
+  display: none;
```

Comment:
We already use `.usa-alert-full-width` alerts for displaying homepage banners, such as for the recent government shutdown. I'm afraid this is going to hide those. I think we need to add another class for Drupal-rendered dismissible alerts. Or possibly, alerts rendered from Drupal can have inline style applied to hide them.

Reply:
@ncksllvn :+1: